msbr_reproc.tex

```diff
@@ -157,17 +157,17 @@ \section{Description of the actual work}
 2 safety rods consisting of boron carbide clad. The under-moderated zone, Zone 
 II, with 37\% fuel salt, and radial reflector, surrounds the Zone I core region 
 and serves to diminish neutron leakage. In this work, 
-only a Zone I element with a $3.42138 cm$ diameter central fuel channel and molten-salt 
+only a Zone I element with a 3.42138 cm diameter central fuel channel and molten-salt 
 fuel volume fraction of 0.132 was considered. Figure~\ref{fig:zoneI} shows the 
 geometry of the unit cell model. The boundary condition of the unit cell is 
-periodic. The density of the fuel salt is $3.3304 g/cm^3$ and the graphite density is 
-$1.843 g/cm^3$. The temperature of fuel and graphite is fixed at 908K for the whole 
+periodic. The density of the fuel salt is 3.3304 g/cm$^3$ and the graphite density is 
+1.843 g/cm$^3$. The temperature of fuel and graphite is fixed at 908K for the whole 
 reprocessing cycle \cite{robertson_conceptual_1971}.
 
 \begin{figure}[htbp!] % replace 't' with 'b' to force it to be on the bottom
         \centering
-        \includegraphics[width=0.90\linewidth]{zone_I_mesh.png}
-        \caption{MSBR unit cell of Zone I geometry.}
+        \includegraphics[width=0.95\linewidth]{zone_I_mesh.png}
+        \caption{MoltenSalt Breeder Reactor (MSBR) unit cell of Zone I geometry.}
```

Comment:
(or, you could use "\glsfirst{MSBR}" which will expand it like it's the first time.)